includes/admin/customers/class-customer-table.php

```diff
@@ -310,10 +310,15 @@ public function prepare_items() {
 
 		$this->total = edd_count_total_customers( $this->args );
 
-		$this->set_pagination_args( array(
-			'total_items' => $this->total,
-			'per_page'    => $this->per_page,
-			'total_pages' => ceil( $this->total / $this->per_page ),
```

Comment:
I'd prefer we just do a strict typing on these to variables instead of making it more confusing by setting to `1` and then changing it later if `per_page` is set, which it will be.

It could even just read `'total_pages' => ceil( (int) $this->total / (int) $this->per_page ),`

Reply:
Hmmm.... I see what you mean.
I pushed a fix, but I think it would still be safer to add a check and make sure that divisor is not `0`. Unless we're absolutely certain there's no chance that `$this->per_page` will ever be zero no matter what users do? If we're sure, then we can just do
```php
'total_pages' => ceil( (int) $this->total / (int) $this->per_page ),
```
as suggested